src/lib/nvme_access.c

```diff
@@ -44,7 +44,7 @@ static int send_recv_nvme_pt_ioctl(int fd, int send, uint8_t proto_id,
 	return status;
 }
 
-static int opal_send(int fd, uint16_t com_id, uint8_t *buf, int buf_len)
```

Comment:
add parameter to opal_send proto_id or somethig and make this function public. 
don't add opal_send_secp02